### PR TITLE
#168095164 Fix password reset link sent to user email

### PR DESCRIPTION
**Pull request by kennyedward**
#### What does this PR do?
This pull request updates the reset link sent to a user email

#### Description of Task to be completed
- Reset and update user password

#### How should this be manually tested?
- Clone and cd into this repository
- run ```npm run start:dev```
- Using _Postman_ make a **POST** request with ```{ email: someone@gmail.com }``` to http://localhost:{Port}/api/v1/auth/reset_password/ to get the reset link
- Then make a **PATCH** request to http://localhost:{Port}/api/v1/auth/update_password/:id/:token
to update the password

#### Any Background Context You want to provide?
The user requesting password reset/update must be a  registered user

#### What are the relevant pivotal tracker stories?

[#168095164](https://pivotaltracker.com/story/show/168095164)

#### Screenshots (if appropriate)
Reste Link
Password Update Success

#### Questions:
N/A

### CODE REVIEW

File: tests/routes/bookmarks.test.js [hunk 1]
```diff
@@ -126,4 +114,33 @@ describe('Bookmark Endpoint', () => {
...
+
+    it('should shpuld', async () => {
+      await Bookmarks.destroy({ where: {} });
+
+      const response = await chai
+        .request(app)
+        .get(`${baseRoute}/articles/bookmarks`)
+        .set('Authorization', `Bearer ${userToken}`);
+
+      response.should.have.status(200);
```
Comment (successgilli) [inline tests/routes/bookmarks.test.js:138]: Nice job adding more tests.

File: src/services/notification.js [hunk 2]
```diff
@@ -320,6 +320,20 @@ class Notification extends EventEmitter {
...
     return this.sendMail('passwordRecovery', payload);
   }
+
+  /**
+   * Handles notification for password update successful
+   *
+   * @function
+   *
+   * @param {Array} payload - An Object that contains the user email
+   *
```
Comment (successgilli) [inline src/services/notification.js:329]: Please, why is there more than one line space before the return definition in the jsdocs of this function?

**Review by successgilli — COMMENTED** [review]
Nice work @kennyedward 
- Please, change the scope of the commit message 'reset password' according to the wiki.
If you must use two words, concatenate them with a hyphen.
- Change the title of the PR to conform to standards on the wiki.

**Comment by kennyedward** [comment]
> Nice work @kennyedward
> 
> * Please, change the scope of the commit message 'reset password' according to the wiki.
>   If you must use two words, concatenate them with a hyphen.
> * Change the title of the PR to conform to standards on the wiki.

Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
For the PR - It's updated.

**Comment by successgilli** [comment]
> > Nice work @kennyedward
> > 
> > * Please, change the scope of the commit message 'reset password' according to the wiki.
> >   If you must use two words, concatenate them with a hyphen.
> > * Change the title of the PR to conform to standards on the wiki.
> 
> Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
> For the PR - It's updated.

File: src/services/notification.js (hunk 2 above)
Comment (kennyedward) [reply src/services/notification.js:329]: Thanks for catching that. It is fixed now.

**Comment by successgilli** [comment]
> > Nice work @kennyedward
> > 
> > * Please, change the scope of the commit message 'reset password' according to the wiki.
> >   If you must use two words, concatenate them with a hyphen.
> > * Change the title of the PR to conform to standards on the wiki.
> 
> Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
> For the PR - It's updated.

**Comment by successgilli** [comment]
> > Nice work @kennyedward
> > 
> > * Please, change the scope of the commit message 'reset password' according to the wiki.
> >   If you must use two words, concatenate them with a hyphen.
> > * Change the title of the PR to conform to standards on the wiki.
> 
> Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
> For the PR - It's updated.

Sorry for closing the PR initially.
I wanted to find an option to refer you to a comment @iverenshaguy  made on @Bastieno's most recent PR, then I hit close.
image

**Comment by iverenshaguy** [comment]
> > Nice work @kennyedward
> > 
> > * Please, change the scope of the commit message 'reset password' according to the wiki.
> >   If you must use two words, concatenate them with a hyphen.
> > * Change the title of the PR to conform to standards on the wiki.
> 
> Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
> For the PR - It's updated.

Actually @kennyedward, 2 words concatenated with a dash. Please update this in the wiki. @successgilli thanks for catching this.

**Comment by kennyedward** [comment]
> > > Nice work @kennyedward
> > > 
> > > * Please, change the scope of the commit message 'reset password' according to the wiki.
> > >   If you must use two words, concatenate them with a hyphen.
> > > * Change the title of the PR to conform to standards on the wiki.
> > 
> > 
> > Thanks for the feedback. However, in the team convention doc, I couldn't locate where it was stated but I found **"It should also be a maximum of 2 words."**
> > For the PR - It's updated.
> 
> Actually @kennyedward, 2 words concatenated with a dash. Please update this in the wiki. @successgilli thanks for catching this.

It is now updated. Thanks, @iverenshaguy @successgilli.

**Review by iverenshaguy — APPROVED** [review]
LGTM

**Pull request merged**